pkg/client/dart/templates/api.dart

```diff
@@ -136,14 +140,13 @@ class APIClient {
 {{- 	end }}
 	}
 
-	Map<String, dynamic> getRequestObject(Map<String, dynamic> obj, List<String> routingPath) {
+	Map<String, dynamic> getRequestObject(Map<String, dynamic> obj, List<String> routingPath, bool isGET) {
     final copied = {...obj};
 
     copied.forEach((key, value) {
       if (routingPath.contains(key))
         copied.remove(key);
-      else
-        copied[key] = value.toString();
+      else if (isGET) copied[key] = value.toString();
```

Comment:
上改行してるのでここも改行したいかもしれない
```suggestion
      else if (isGET)
        copied[key] = value.toString();
```

Reply:
flutter formatの結果なんですよねw
どうせformatterかけられるしTemplateにはわかりやすく改行させた方がいいかな？

https://user-images.githubusercontent.com/30588003/152099895-6a9bec15-5fdc-41d0-94c6-6d8afa890e67.mov

Reply:
なるほど！？
そのフォーマット理論はよくわからないけどformatterがそうならOKです

Reply:
上の方はむしろ一行から二行に変えられてしまって、下の方は逆というねw
フォーマット理論よく分からない分かる